packages/terra-dropdown-button/src/_DropdownList.jsx

```diff
@@ -160,11 +171,24 @@ class DropdownList extends React.Component {
    * @return {Array<React.ReactNode>} the array of children
    */
   cloneChildren() {
-    return React.Children.map(this.props.children, (child, index) => React.cloneElement(child, {
-      isActive: index === this.state.active,
-      requestClose: this.props.requestClose,
-      'data-terra-dropdown-list-item': true,
-    }));
+    return React.Children.map(this.props.children, (child, index) => {
+      const currentItemLabel = this.props.children[index]?.props.label;
+      const currentIndex = index + 1;
+      const ofText = this.props.intl.formatMessage({ id: 'Terra.dropdownButton.of' });
+      const totalItems = this.props.children.length;
+      let ariaLabel = null;
+      if (SharedUtil.isMac() && currentIndex === 1 && totalItems) {
```

Comment:
since we will be removing aria-expanded on [DropdownButton.jsx ](https://github.com/cerner/terra-core/blob/1adf2f2ea4c8e34b823065374c5ac1ba7ad8f025/packages/terra-dropdown-button/src/DropdownButton.jsx#L209) . we should remove this condition of MAC and make it consistent across all the screen readers and browsers.

Reply:
Implemented and tested it is working as expected.